TestsFoundation/TestsFoundation/Extensions/XCUIElementExtensions.swift

```diff
@@ -19,50 +19,241 @@
 import Foundation
 import XCTest
 
+public var app: XCUIApplication { XCUIApplication() }
+
 public extension XCUIElement {
-    func find(label: String, type: XCUIElement.ElementType = .any) -> Element {
-        return descendants(matching: type).matching(label: label).firstElement
+    static let defaultTimeout: TimeInterval = 15
+    static let defaultGracePeriod: UInt32 = 1
+    var isVisible: Bool { exists }
+    var isVanished: Bool { !(exists && isHittable) }
+    func tacticalSleep(ms: UInt32 = 500000) { usleep(ms) }
+
+    enum ElementCondition {
+        case visible
+        case vanish
+        case value
+        case label
+        case enabled
+        case selected
+        case unselected
+        case hittable
+        case labelContaining
+        case labelHasPrefix
+    }
+
+    enum ElementAction {
+        case swipeUp
+        case swipeDown
+        case swipeRight
+        case swipeLeft
+        case tap
+        case showKeyboard
+        case hideKeyboard
+        case pullToRefresh
+    }
```

Comment:
The order we aim to use is to have type definitions first, followed by static vars then private vars and finally the functions.

Reply:
Done.